change/@fluentui-react-a941c131-ba0e-40bc-a4b1-a87cd8554cff.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28907&commitId=0b1a52b7b3b398f22e0f0f408eafbf1a1412e282&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.customSize+image.normal.chromium.png|84| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3357| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|52| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|1340| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>